src/Command/CommandBase.php

```diff
@@ -109,6 +108,19 @@ public function __construct(
     $this->setLocalDbName();
     $this->setLocalDbHost();
     parent::__construct();
+    if ($this->commandRequiresAuthentication()) {
+      $this->appendHelp('This command requires authentication via the Cloud Platform API.');
+    }
+    if ($this->commandRequiresDatabase()) {
+      $this->appendHelp('This command requires an active database connection. Set the following environment variables prior to running this command: '
+        . 'ACLI_DB_HOST, ACLI_DB_NAME, ACLI_DB_USER, ACLI_DB_PASSWORD');
```

Comment:
This could be the compromise we want. But aren't we going to get support tickets on how to set `ACLI_DB_HOST, ACLI_DB_NAME, ACLI_DB_USER, ACLI_DB_PASSWORD`? Should we further clarify that the command shouldn't typically be used outside of an IDE?